docs/guide/server.md

```diff
@@ -0,0 +1,122 @@
+# Using Dolos Server
+
+We provide a free online service to run Dolos directly in our browser.
+You can use this service at [dolos.ugent.be/server](https://dolos.ugent.be/server).
+
+If you want to follow along, you can download a [ZIP with sample files](/simple-dataset.zip) which you can analyze on our server.
+
+## Upload and analyze files
+
+![Screenshot of the upload form](/images/screenshots/dolos-upload.png)
+
+To start a plagiarism analysis, upload a ZIP-file containing the files you want to analyze.
+You can specify a name for your analysis, which will be used to identify it in the report history.
+
+Additionally, you can select the programming language of the files.
+Dolos will automatically detect the language based on the file extension, but you can override this if needed.
+
+Click "Analyze" to upload your files and start the analysis.
+
+
+### Upload format
+
+Currently, we only support uploading a single ZIP-file containing all the files you want to analyze with an required `info.csv` file.
+This is the format in which [Dodona](./dodona.md) exports submissions.
```

Comment:
I would put this in a "tip" box and mention this is for UGent users because others might be confused about this.